pkg/driver/executor/executor.go

```diff
@@ -530,23 +530,26 @@ func (ex *Executor) getOrCreatePort(_ context.Context, machineName string) (stri
 }
 
 func (ex *Executor) deletePort(_ context.Context, machineName string) error {
-	portID, err := ex.Network.PortIDFromName(machineName)
+	portList, err := ex.Network.ListPorts(ports.ListOpts{
+		Name: machineName,
+	})
 	if err != nil {
-		if client.IsNotFoundError(err) {
-			klog.V(3).Infof("port [Name=%q] was not found", machineName)
-			return nil
-		}
 		return fmt.Errorf("error deleting port [Name=%q]: %s", machineName, err)
 	}
+	if len(portList) == 0 {
+		klog.V(3).Infof("port [Name=%q] was not found", machineName)
+		return nil
+	}
 
-	klog.V(2).Infof("deleting port [Name=%q]", machineName)
```

Comment:
The logging on verbose level 2 has been removed by intention?

Reply:
I added it back.